Automattic/wp-calypso
# Add QueryGSuiteUsers Component

#### Changes proposed in this Pull Request

* Replaces existing Google Apps fetch actions with a GSuiteUsersQuery Component and related actions

#### Testing instructions
Overview: Run through all aspects of the G Suite User management experience. Management, adding users, etc.

1. Navigate to `/domains/manage/email/:siteSlug` for a site with G Suite
1. Compare to WordPress.com: confirm that all users were correctly loaded.
1. Click "Add G Suite User" for an existing G Suite Subscription
1. Attempt to add an existing user and confirm that it fails with the "You already have this email address" error. <img width="781" alt="Screen Shot 2019-03-18 at 4 21 11 PM" src="https://user-images.githubusercontent.com/2810519/54569925-dc846980-4999-11e9-841d-9590c19a5605.png">
1. Add a user with a new mailbox
1. Confirm that the new user is added to `/domains/manage/email/:siteSlug`
1. Run Tests with this command ` npm run test-client client/state/data-layer/wpcom/gsuite-users/ client/state/gsuite-users/`
